api/notd/model.py

```diff
@@ -514,3 +514,28 @@ class GmCollectionRow:
 class LatestAccountGm:
     accountGm: AccountGm
     accountCollectionGms: List[AccountCollectionGm]
+
+
+@dataclasses.dataclass
+class RetrievedCollectionOverlap:
+    registryAddress: str
+    galleryAddress: str
+    ownerAddress: str
+    tokenCount: int
+    galleryCount: int
+
+
+@dataclasses.dataclass
+class CollectionOverlap(RetrievedCollectionOverlap):
+    collectionOverlapId: int
+    createdDate: datetime.datetime
+    updatedDate: datetime.datetime
+
+
+@dataclasses.dataclass
+class GalleryCollectionOverlap:
+    galleryAddress: str
+    registryAddress: str
+    ownerCount: str
+    tokenCount: int
+    collectionOverlaps: List[CollectionOverlap]
```

Comment:
this structure doesnt make sense to me - what are the fields here, they look like duplicates of the fields in the listed object